Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/reference/config.adoc

= Configuration
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Core configuration covers the command line, env variables and configuration options.

Reviewer: ```suggestion
:description: Covers the different configuration options of Lisk Core, including env variables.
```
